docs/dictionary/function/replaceText.lcdoc

```diff
@@ -60,13 +58,12 @@ the <matchExpression> to make the match case-insensitive.)
 >*Note:* A number of characters in regular expressions have special
 > meanings and need to be escaped with back slashes For example period
 > (".") matches any character, so in order to replace period characters
-> with a regular expression use "\." . For more information on regular
+> with a <regular expression> use "\." . For more information on regular
 > expressions see the Perl documentation here:
-> &lt;http://perldoc.perl.org/perlre.html&gt; 
-
-&nbsp;
+> <http://perldoc.perl.org/perlre.html> 
```

Comment:
@asayd URLs don't need to be in angle brackets, they automatically render as links. You can also use `[link text](url)`.